### PR TITLE
build!: minimum supported Node.js version is 18

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

Minimum supported Node.js version is 18

### Breaking Changes

Yes

### Additional Info

No

### CODE REVIEW

**Comment by ricardogobbosouza** [comment]
The error that occurs in ci is that the action `wagoid/commitlint-github-action@v5` is not using the latest version of `@commitlint/lint`

The latest version added a new rule

https://github.com/conventional-changelog/commitlint/releases/tag/v18.6.0

**Comment by alexander-akait** [comment]
> The error that occurs in ci is that the action wagoid/commitlint-github-action@v5 is not using the latest version of @commitlint/lint

Ignore it

File: babel.config.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = (api) => {
         '@babel/preset-env',
         {
           targets: {
-            node: '14.15.0',
+            node: '18.0.0',
```
Comment (alexander-akait) [inline babel.config.js:13]: `node: "18.12.0",`

**Review by alexander-akait — COMMENTED** [review]
Let's use the first LTS version

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
@ricardogobbosouza do you have premission to setup Github Actions?

**Comment by ricardogobbosouza** [comment]
> @ricardogobbosouza do you have premission to setup Github Actions?

Yes I have.

**Pull request merged**